xray/src/backend.rs

```diff
@@ -109,82 +122,88 @@ struct HandleNodesForLevel {
     meta: Arc<Meta>,
 }
 
+pub fn get_nodes_for_level(
```

Comment:
what about putting this as a member function of `Meta` in `point_cloud_viewer/xray/src/lib.rs`?
Also, this could be named `get_nodes_in_frustum_for_level()` or something similar.

Reply:
Ok, I will make it a method. As discussed I will just leave the name as is for now, as I think we want to change the implementation to take a bounding box rather than a frustum.